src/library/controllers/message/interface.js

```diff
@@ -58,146 +58,168 @@ export default (options = {}) => ({
         });
 
         return awaitTreatments.then(() =>
-            ZalgoPromise.all(
-                validContainers.map(container => {
-                    try {
-                        const merchantOptions = objectMerge(
-                            getGlobalState().config,
-                            objectMerge(options, getInlineOptions(container))
-                        );
-
-                        if (!container.hasAttribute('data-pp-id')) {
-                            container.setAttribute('data-pp-id', nextIndex());
-                        }
+            /* eslint-disable-next-line  promise/no-native, compat/compat */
+            Promise.all(
+                validContainers.map((container, mapIndex) => {
+                    /* eslint-disable-next-line  promise/no-native, compat/compat, consistent-return */
+                    return new Promise((resolve, reject) => {
```

Comment:
Based on the [previous comment](https://github.com/paypal/paypal-messaging-components/pull/939#discussion_r1218588949) we can't make this change?
```suggestion
            ZalgoPromise.all(
                validContainers.map((container, mapIndex) => {
                    return new ZalgoPromise((resolve, reject) => {
```

Reply:
When I removed the promises for zalgopromise I forgot to add back the new ZalgoPromise((resolve, reject) portion. Thats why I couldn't get it to return